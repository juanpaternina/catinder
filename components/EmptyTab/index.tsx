import Colors from "@/constants/Colors";
import { Text, View, StyleSheet } from "react-native";

type Props = {
  title: string;
};

export default function EmptyTab({ title }: Props) {
  return (
    <View style={styles.container}>
      <Text style={styles.title}>{title}</Text>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: "center",
    paddingTop: 200,
    backgroundColor: Colors.default.background,
  },
  title: {
    fontSize: 126,
    color: "#BFBFC0",
  },
});
