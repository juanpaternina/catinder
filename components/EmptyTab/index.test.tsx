import { render, screen } from "@testing-library/react-native";
import EmptyTab from "./index";

describe("EmptyTab", () => {
  it("renders correctly", () => {
    render(<EmptyTab title="Test" />);
    expect(screen.getByText("Test")).toBeTruthy();
    expect(screen).toMatchSnapshot();
  });
});
