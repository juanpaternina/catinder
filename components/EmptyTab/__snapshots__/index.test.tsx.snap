// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyTab renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#FBFAFF",
      "flex": 1,
      "paddingTop": 200,
    }
  }
>
  <Text
    style={
      {
        "color": "#BFBFC0",
        "fontSize": 126,
      }
    }
  >
    Test
  </Text>
</View>
`;
